docs/development/observability/logging.mdx

```diff
@@ -10,12 +10,24 @@ The `SERVER_VERBOSITY` variable accepts the following values: `DEBUG`, `INFO`, `
 
 ## Server logging format
 
+You can customize the format by setting the `SERVER_LOGGING_FORMAT` environment variable. By default it is set as
+`SERVER_LOGGING_FORMAT=plaintext`. If it is set to unknown format it will use default value instead.
+
+### Plaintext format
+
+The plaintext logging format will be used by default. Or it can be set explicitly by setting the
+`SERVER_LOGGING_FORMAT=plaintext` environment variable.
+
 By default, the server will log messages in the following format:
 ```
-[%(levelname)s]:[%(name)s]:%(message)s
+%(levelname)s:%(name)s:%(message)s
 ```
 
-You can customize the format by setting the `SERVER_LOGGING_FORMAT` environment variable.
+It can be customized by setting the `SERVER_LOGGING_FORMAT` environment variable.
+Example:
+```
+SERVER_LOGGING_FORMAT=%(asctime)s\t[%(name)25.25s]\t%(levelname)5s: %(message)s
```

Comment:
Yeah, it should be enveloped in ${} to use raw special symbols. I can either remove special symbols or test different variations and come back with correct version (my guess is that it just should be `SERVER_LOGGING_FORMAT=${%(asctime)s\t[%(name)25.25s]\t%(levelname)5s: %(message)s}`)

Reply:
So yeah - correct syntax is `SERVER_LOGGING_TEMPLATE=$'%(asctime)s\t\t[%(name)25.25s]\t%(levelname)5s: %(message)s'`

The only case in which it doesn't work is .env file. Didn't find any solutions to the problem where special characters are escaped if you pass env variable through .env file. So looks like it just isn't possible.

My proposition - explicitly describe in docs. That you can pass it as env variable using syntax above. BUT you can't do it using .env file because special charaters like tabulation and newline will be escaped.

Reply:
Done